packages/phoenix-otel/src/phoenix/otel/otel.py

```diff
@@ -38,6 +38,7 @@ def register(
     batch: bool = False,
     set_global_tracer_provider: bool = True,
     headers: Optional[Dict[str, str]] = None,
+    protocol: Optional[Literal["http", "grpc"]] = None,
```

Comment:
I think we should probably match the protocol names here https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_protocol

> Valid values are:

grpc to use OTLP/gRPC
http/protobuf to use OTLP/HTTP + protobuf
http/json to use OTLP/HTTP + JSON